libnavui-androidauto/src/main/java/com/mapbox/androidauto/car/placeslistonmap/PlacesListOnMapLayerUtil.kt

```diff
@@ -12,6 +13,7 @@ import com.mapbox.maps.extension.style.sources.generated.GeoJsonSource
 import com.mapbox.maps.extension.style.sources.generated.geoJsonSource
 import com.mapbox.maps.extension.style.sources.getSource
 
+@UiThread
```

Comment:
Where does this requirement come from?

Reply:
It interacts with `Style`. From docs to `Style` class:
```
Note: Similar to a View object, a Style should only be read and modified from the main thread.
```